api/api/util.py

```diff
@@ -254,26 +252,26 @@ def _create_problem(exc: Exception, code=None):
         ProblemException or `exc` exception type
     """
     ext = None
-    if isinstance(exc, WazuhException):
+    if isinstance(exc, exception.WazuhException):
         ext = remove_nones_to_dict({'remediation': exc.remediation,
                                     'code': exc.code,
                                     'dapi_errors': exc.dapi_errors if exc.dapi_errors != {} else None
                                     })
 
-    if isinstance(exc, WazuhInternalError):
-        raise ProblemException(status=500 if not code else code, type=exc.type, title=exc.title, detail=exc.message,
-                               ext=ext)
-    elif isinstance(exc, WazuhPermissionError):
+    if isinstance(exc, (exception.WazuhInternalError, exception.WazuhClusterError)):
```

Comment:
If `WazuhClusterError` inherited from `WazuhInternalError` we could reduce this, in addition to some other code improvements on other modules.

Reply:
Agree